test/blackbox-tests/test-cases/path-variables/run.t

```diff
@@ -0,0 +1,56 @@
+dune files
+==========
+
+%{dep:string}
+-------------
+
+In expands to a file name, and registers this as a dependency.
+
+  $ dune build --root dune @test-dep
+  Entering directory 'dune'
+  dynamic-contents
+
+%{path-no-dep:string}
+---------------------
+
+This form does not exist, but displays an hint:
+
+  $ dune build --root dune-invalid @test-path-no-dep
+  Entering directory 'dune-invalid'
+  File "dune", line 7, characters 17-54:
+  Error: The ${path-no-dep:file-that-does-not-exist} syntax has been removed from dune.
```

Comment:
I'm not entirely sure, but shouldn't this say `The ${path:...} syntax  has been removed from dune.`, i.e. without the actual payload. It's seems to me that if we print the actual payload, we are talking about this instance of the syntax rather than the syntax in general. What do you think?

Reply:
That's a good idea. The payload probably only makes sense for suggestions/corrections.